ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -122,6 +135,25 @@ public static String getInstanceName(String host, Integer port) {
     return dataCenterToZkAddress;
   }
 
+  /**
+   * Create a instance of {@link HelixPropertyStore} based on given {@link HelixPropertyStoreConfig}.
+   * @param zkServers the ZooKeeper server address.
+   * @param propertyStoreConfig the config for {@link HelixPropertyStore}.
+   * @param subscribedPaths a list of paths to which the PropertyStore subscribes.
+   * @return the instance of {@link HelixPropertyStore}.
+   * @throws IllegalArgumentException
+   */
+  public static HelixPropertyStore<ZNRecord> createHelixPropertyStore(String zkServers,
+      HelixPropertyStoreConfig propertyStoreConfig, List<String> subscribedPaths) throws IllegalArgumentException {
```

Comment:
you need not explicitly declare `throws IllegalArgumentException` since it is a `RuntimeException`